Backend/ssointegration /src/main/java/com/ssointegration/config/CorsConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.ssointegration.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class CorsConfig implements WebMvcConfigurer {
```

Comment:
why are you configuring the cors twice ? i.e in CorsConfig and in WebConfig, just do it in either one of them and remove the other